content/docs/command-reference/repro.md

```diff
@@ -13,7 +13,7 @@ usage: dvc repro [-h] [-q | -v] [-f] [-s] [-m] [--dry] [-i]
                  [targets [<target> ...]]
 
 positional arguments:
-  targets       Stages to reproduce. 'dvc.yaml' by default.
+  targets       Stages or output files to reproduce. 'dvc.yaml' by default.
```

Comment:
```suggestion
  targets       Stages or outputs to reproduce. 'dvc.yaml' by default.
```

Reply:
Actually, not sure about using the term "output" so much, per https://github.com/iterative/dvc.org/pull/2357#pullrequestreview-629433013... Will comment in the core PR.

Reply:
So let's apply the same as in https://github.com/iterative/dvc/pull/5273#pullrequestreview-630480437 here please, @Vonski. Thanks!